docs/tutorials/openfoam/openfoam.md

```diff
@@ -0,0 +1,5 @@
+# Cluster Toolkit - OpenFOAM Simulation and Visualization
```

Comment:
If you name the file `README.md` instead, it would be shown nicely on GitHub web page on directory view page.